lib/linear_dispatcher.rb

```diff
@@ -8,19 +8,10 @@ def next_pending_approval(proposal)
     end
   end
 
-  def email_next_pending_approver(proposal)
-    if approval = self.next_pending_approval(proposal)
-      self.email_approver(approval)
-    end
-  end
-
-  def deliver_new_proposal_emails(proposal)
-    self.email_next_pending_approver(proposal)
-    super
-  end
-
   def on_approval_approved(approval)
-    self.email_next_pending_approver(approval.proposal)
+    if next_approval = self.next_pending_approval(approval.proposal)
+      self.email_approver(next_approval)
+    end
```

Comment:
I don't think `self.email_approver(next_approval)` is less descriptive than `self.email_next_pending_approver`. I can change the variable name to `next_pending_approval` if you really want?

Removing this indirection function reduces the surface area of this class, which improves encapsulation. When refactoring in the future, we won't need to see "is anyone calling this method" before changing it.